src/events.rs

```diff
@@ -312,3 +312,11 @@ pub enum VirtualKeyCode {
     WebStop,
     Yen,
 }
+
+#[derive(Default, Debug, Clone, Copy)]
+pub struct Mods {
```

Comment:
This needs some documentation. Not everybody knows what a modifier is.
Also the fields should be 4-spaces indented, not 2.